q2_diversity_lib/tests/test_alpha.py

```diff
@@ -234,3 +239,42 @@ def test_do_not_drop_undefined_samples(self):
         actual = shannon_entropy(table=self.input_table,
                                  drop_undefined_samples=False)
         pdt.assert_series_equal(actual, self.expected)
+
+
+class AlphaPassthroughTests(TestPluginBase):
+    package = 'q2_diversity_lib.tests'
+
+    def setUp(self):
+        super().setUp()
+        self.method = self.plugin.actions['alpha_passthrough']
+        self.available_metrics = alpha.METRICS['NONPHYLO']['UNIMPL']
```

Comment:
Seems like unneccessary indirection to me - `METRICS` is already a "package constant", I don't think giving it an alias improves the clarity of the tests in this suite.

Reply:
handled here, in `test_beta`, and in `test_beta_phylogenetic`